dewy/config.py

```diff
@@ -79,8 +25,31 @@ def custom_generate_unique_id_function(route: APIRoute) -> str:
     return convert_snake_case_to_camel_case(route.name)
 
 
-def _get_config(request: Request) -> Config:
+APP_CONFIGS: dict[str, Any] = {
+    "title": "Dewy Knowledge Base API",
+    "version": "0.3.0",
```

Comment:
Does this need to be added to the list of things to change during release?

Reply:
It's already in the list -- it just moved where it was within this file.